hack/verify-py.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+ROOTDIR=$(git rev-parse --show-toplevel)
+REQ_FILE=${ROOTDIR}/requirements.txt
+VENV="/tmp/venv"
+PYLINT="pylint"
+
+if ! command -v ${PYLINT} &>/dev/null; then
+    
+    check="$(python3 -m pip install -r ${REQ_FILE} 2>&1 | { grep 'Permission denied' || true; })"
```

Comment:
Why not just use the virtualenv all the time?

Reply:
The pylint installation worked fine directly in RHEL dev VM, without virtualenv. And as virtualenv will put a little overhead in the VM, I think we should do that only for CI.

Reply:
Yes, we need all the packages listed in `requirements.txt`  for pylint to run successfully, otherwise it will throw `(import-error)`. 

For having venv locally, we could setup that in `_output/` folder and install all the dependencies there. In that case, we should run all python screipts from that venv, and install further dependencies there only. I should not have 2 copies of dependencies.

Reply:
As discussed in Slack, we may later on, create a separate PR to run all .py scripts from venv rather than requiring to have deps installed on system (usually /usr/, and not $HOME...)

Reply:
It's not really recommended practice to use pip to install things directly into the system python space, so I would prefer we use virtualenv all the time.

Reply:
Okay, I've updated the script to create a virtual environment all the time.